packages/bpk-component-close-button/src/BpkCloseButton.module.scss

```diff
@@ -22,13 +22,7 @@
 $button-padding: tokens.bpk-spacing-sm() * 0.5;
 
 .bpk-close-button {
-  width: tokens.bpk-spacing-lg();
   height: tokens.bpk-spacing-lg();
-
-  // The margin here is used to ensure that the icon is aligned to
-  // the leading or trailing edge of its parent container
-  margin-right: -1 * $button-padding;
-  margin-left: -1 * $button-padding;
```

Comment:
We always set a margin on the button when it's not always wanted. We overrode this in the BpkModal [here](https://github.com/Skyscanner/backpack/pull/3416/files#diff-5dd199125d8d57227be7e5e9a0633b3b60d925e37378f67cd2ffb86e881ea65dL111)

Reply:
It's best to leave the button without a margin and allow the other componets to add one if needed.